src/GitHub.TeamFoundation.14/Base/TeamExplorerServiceHolder.cs

```diff
@@ -162,11 +165,11 @@ void UIContextChanged(object sender, UIContextChangedEventArgs e)
                         // and try again. See issue #23
                         if (repos == null)
                         {
-                            VsOutputLogger.WriteLine(string.Format(CultureInfo.CurrentCulture, "Error 2001: ActiveRepositories is null. GitService: '{0}'", GitService));
+                            log.Error("Error 2001: ActiveRepositories is null. GitService: '{GitService}'", GitService);
```

Comment:
Should we avoid using `'` with Serilog? I know quoting strings is unnecessary. Another a few lines below.